M2/Macaulay2/m2/startup.m2.in

```diff
@@ -398,6 +398,8 @@ usage := arg -> (
      << "    --stop             exit on error" << newline
      << "    --texmacs          TeXmacs session mode" << newline
      << "    --version          print version number and exit" << newline
+     << "    --webapp           WebApp session mode" << newline
```

Comment:
It's probably better to merge "master" into your branch rather than "development", because "master" has been tested overnight.

Reply:
But you set the target of this branch as development, which caused a merge conflict. How should we resolve this?

Reply:
Well, hmm, one could first try merging with "master" -- that might resolve the conflict.  If not and the conflict is provoked by a recent pull request from someone else, wait a day and see if it finds its way into "master" or gets reverted, due to failing the overnight test.